RELEASE-NOTES.md

```diff
@@ -19,6 +19,21 @@ The Axom project release numbers follow [Semantic Versioning](http://semver.org/
 
 ## [Unreleased] - Release date yyyy-mm-dd
 
+### Added
+
+### Changed
+
+### Removed
+
+
+## [Version 0.10.0] - Release date 2024-09-xx
```

Comment:
Please update the links for `axom@0.10` and `axom@0.9` at the bottom of this file

Reply:
Done.